src/cascadia/TerminalApp/SuggestionsControl.cpp

```diff
@@ -1063,27 +1188,16 @@ namespace winrt::TerminalApp::implementation
         const auto proposedX = gsl::narrow_cast<int>(_anchor.X - 40);
         // If the control is too wide to fit in the window, clamp it fit inside
         // the window.
-        const auto maxX = gsl::narrow_cast<int>(space.Width - actualSize.width);
+        const auto maxX = gsl::narrow_cast<int>(space.Width - ActualWidth());
         const auto clampedX = std::clamp(proposedX, 0, maxX);
 
-        // Create a thickness for the new margins
+        // Create a thickness for the new margins. This will set the left, then
+        // we'll go update the top.
         auto newMargin = Windows::UI::Xaml::ThicknessHelper::FromLengths(clampedX, 0, 0, 0);
-        // Now, position vertically.
-        if (_direction == TerminalApp::SuggestionsDirection::TopDown)
-        {
-            // The control should open right below the cursor, with the list
-            // extending below. This is easy, we can just use the cursor as the
-            // origin (more or less)
-            newMargin.Top = (_anchor.Y);
-        }
-        else
-        {
-            // Position at the cursor. The suggestions UI itself will maintain
-            // its own offset such that it's always above its origin
-            newMargin.Top = (_anchor.Y - actualSize.height);
-        }
         Margin(newMargin);
```

Comment:
Do we need this call anymore since we're calling `_recalculateTopMargin()` right after it?

Reply:
Yea, we're using this to set the left margin to where the cursor is. `_recalculateTopMargin` just sets the top, which varies based on the selected item, while the left margin doesn't.